docs/code-of-conduct.rst

```diff
@@ -0,0 +1,33 @@
+Code of Conduct
+===============
+
+Like the technical community as a whole, the Write the Docs team and community is made up of a mixture of professionals and volunteers from all over the world, working on every aspect of the mission - including mentorship, teaching and connecting people.
+
+Diversity is one of our huge strengths, but it can also lead to communication issues and unhappiness. To that end, we have a few ground rules that we ask people to adhere to when they're participating within this community and project. These rules apply equally to founders, mentors, admins, and those seeking conversation, discussion, help, and guidance.
```

Comment:
I think acknowledging that diversity isn't free is useful -- I think "Diversity is one of our huge strengths, but it can also lead to communication issues" is fine -- the unhappiness part reads a bit weird.